### ISSUE TITLE
Feature Request: Integration as a Terraform provider plugin for RFDK

### ISSUE THREAD

**Issue by queglay** [aws/aws-rfdk#439]
I think it would be amazing to see the RFDK become part of the AWS terraform provider plugins.

### Use Case

I currently configure Deadline in an automated fashion for my Open Firehawk project, and Terraform is my preferred method of orchestration.  Getting the RFDK into Terraform would be incredible.  I would migrate my existing workflows to this if it was possible.

I also created a ticket under the AWS terraform provider repo for this ability.

https://github.com/hashicorp/terraform-provider-aws/issues/19477

---

This is a :rocket: Feature Request

**Comment by ddneilson** [comment]
Thanks for the feature request, Andrew. We've been hearing the same from some of our enterprise customers as well, and it's on our roadmap to look into feasibility. 

Going through your comments on the terraform-provider-aws ticket, it sounds like your ideal solution would be something along the lines of a set of Terraform HCL providers that can be used to setup Deadline. Allowing you to leverage other, existing, providers for filesystems, networking, etc.

**Comment by queglay** [comment]
That's right, Thankyou for hearing me out.  So far HCL is about as close to standardised glue for more systems than anything else out there.  There are even some providers to configure routers with it.

I'd love to see the project make this possible.
